functions/src/aqi-calculation/invalid-aqi-errors.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Errors that can occur during the AQI calculation process. Any of these errors
+ * will result in a sensor being invalid.
+ *
+ * - `InfiniteAqi`- If the calculated AQI for a sensor is infinite.
+ * - `NotEnoughNewReadings` - If not enough new readings were received for a
+ *   sensor in the last 3 hours. Two of the three most recent hours must have at
+ *   least 23 new (and valid) readings. This error only reflects if not enough
+ *   new readings are being received. This error can also occur soon after a
+ *   sensor is activated if not enough new readings are stored in the PM2.5
+ *   buffer yet.
```

Comment:
This description seems adequate enough to ignore my previous comment about creating a new error for new sensors/